### ISSUE TITLE
Network plugin off state

### ISSUE THREAD

**Issue by andyrichardson** [jaagr/polybar#622]
I'm looking to have 3 states for the network module (when using the wifi interface).

These three states would be: wifi connected, wifi disconnected and wifi off. 

As of present, the former two are already implemented but I can't see a means for an 'interface off/disabled' state. Are there any methods for handling this?

Example:

    label-connected = Connected to network
    label-disconnected = Not connected to network
    label-off = Wifi Disabled   # This would be when wifi not enabled

**Comment by patrick96** [comment]
Yeah, the distinction between not connected and not enabled would be nice. I'll have a look at this in the near future.

**Comment by fourtytwo** [comment]
Also, I wonder if it is somehow possible to have only either LAN or WIFI displayed, not both. Right now when I am connected via lan it still tells me that WIFI is connected eventhough i am using lan

**Comment by patrick96** [comment]
@fourtytwo Well, both LAN and Wifi can be connected at the same time, so it makes sense that both tell you they're connected. Since those are separate modules it's not possible to disable one depending on the state of the other

**Comment by fourtytwo** [comment]
yea i probably worded that wrong. but I figured one needed to write some special script to be able to have an xor between the two.

**Comment by patrick96** [comment]
Yeah having only one of the module displayed can't be done by polybar alone. I'm not even sure if it is possible without restarting the bar
